recipes/bashlex/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "bashlex" %}
+{% set version = "0.11" %}
+{% set sha256 = "ea59773824fd8c58fd0f140154f794f3bcb3598d8871f5597631ba36581fe998" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/idank/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
probably a candidate for `noarch: python`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages